test/Image1.rb

```diff
@@ -285,6 +332,16 @@ def test_border
     assert_raise(FreezeError) { @img.border!(2, 2, 'red') }
   end
 
+  def test_capture
+    # assert_nothing_raised { Magick::Image.capture }
```

Comment:
These need some attention in another PR?

Reply:
Yep, seems Magick::Image.capture does not work... It needs to fix another PR.

Reply:
Wonder how you can fix this. The method calls `display=XOpenDisplay(image_info->server_name);` and I don't think that will work in a CI build.

Reply:
Filed issue in https://github.com/rmagick/rmagick/issues/560

Reply:
> Wonder how you can fix this. The method calls display=XOpenDisplay(image_info->server_name); and I don't think that will work in a CI build.

Ah, i see